.github/workflows/scale-test-5-gce.yaml

```diff
@@ -0,0 +1,394 @@
+name: 5 Nodes Scale Test (scale-5)
+
+on:
+  schedule:
+    - cron: '39 0 * * 1-5'
```

Comment:
It probably makes sense to schedule this at a different time vs. the 100-node test.

Reply:
Yes, that seems like a good idea. :+1: